examples/swapi/src/Feed.re

```diff
@@ -0,0 +1,75 @@
+let ste = ReasonReact.stringToElement;
+
+module CharacterQuery = [%graphql
+  {|
+    query films($first: Int) {
+      allFilms(first: $first) {
+        films {
+          id
+          title
+          releaseDate
+        }
+      }
+    }
+|}
+];
+
+module Query = Client.Instance.Query;
+
+let parseFilms = allFilms =>
```

Comment:
maybe would be better to have a bind operator here (>>=) to avoid switch-hadouken

Reply:
How would you go about implementing this? Can you show me?

Reply:
more or less like this:
```reason
module Infix = {
  let (>>=) = (value, fn) =>
    switch value {
    | Some(value) => fn(value)
    | None => None
    };
  let (|?) = (value, default) => 
  switch arg {
  | None => default
  | Some(value) => value
  };

};

...
open Infix;
let parseFilm = film => {
 film 
  >>= (film => List.join(" - ", [film##title |? "", film##releaseDate |? ""]))
 |? "Error retrieving film"
};
let parseFilms = allFilms => {
  allFims
    >>= (allFilms => allFilms##films)
    >>= (films => Some(Array.map(parseFilm, films)))
   |? [||i]
}
```

Reply:
Wow I didn't know that was possible with reason! Thanks so much for the tip, I will try this out, will definitely make the code more readable

Reply:
@excitement-engineer lot of that style here: https://github.com/ncthbrt/reason-nact

Reply:
Cool! Good link @idkjs! I will be sure to include some of the patterns in that link to the updates on the example. Thanks

Reply:
In case anyone is curious, [here's where Nact defines the operators that @idkjs referred to](https://github.com/ncthbrt/reason-nact/blob/master/src/Nact.re#L425): 

```reason
module Operators = {
  let (<-<) = (actorRef, msg) => dispatch(actorRef, msg);
  let (>->) = (msg, actorRef) => dispatch(actorRef, msg);
  let (<?) = (actor, (f, timeout)) => query(~timeout, actor, f);
};
```